src/platform/plugins/private/links/server/content_management/links_storage.ts

```diff
@@ -172,46 +169,30 @@ export class LinksStorage {
     return value;
   }
 
-  async update(
-    ctx: StorageContext,
-    id: string,
-    data: LinksAttributes,
-    options: LinksUpdateOptions
```

Comment:
Tags are still passed in as `options.references` when saving changes to a Links library item from the "View details" action in the Visualize library listing page . I think this is still the only way to add tags to Links library items. We could consider adding `tags` to the incoming and outgoing `LinksState` (like we do with [Dashboard](https://github.com/elastic/kibana/pull/214788)). But the listing page editor probably still needs to support sending references on update.

Reply:
Thanks for pointing this out. `update` route has been restored to pass tags references in b5ee258367edfac0d9178efc8c32db877591b485